docs/howto/windows.txt

```diff
@@ -4,27 +4,27 @@ How to install Django on Windows
 
 .. highlight:: doscon
 
-This document will guide you through installing Python 3.7 and Django on
+This document will guide you through installing Python 3.8 and Django on
 Windows. It also provides instructions for installing `virtualenv`_ and
 `virtualenvwrapper`_, which make it easier to work on Python projects. This is
 meant as a beginner's guide for users working on Django projects and does not
 reflect how Django should be installed when developing patches for Django
 itself.
 
-The steps in this guide have been tested with Windows 7, 8, and 10. In other
+The steps in this guide have been tested with Windows 10. In other
 versions, the steps would be similar. You will need to be familiar with using
 the Windows command prompt.
 
 Install Python
 ==============
 
 Django is a Python web framework, thus requiring Python to be installed on your
-machine. At the time of writing, Python 3.7 is the latest version.
+machine. At the time of writing, Python 3.8 is the latest version.
 
 To install Python on your machine go to https://python.org/downloads/. The
 website should offer you a download button for the latest Python version.
 Download the executable installer and run it. Check the boxes next to ``Install
-launcher for all users (recommended)`` and ``Add Python 3.7 to PATH`` then
+launcher for all users (recommended)`` and ``Add Python 3.8 to PATH`` then
```

Comment:
Add to PATH is evil no? 🙂

I **never** click this. I use the `py` launcher to pick a version, plus `venv` to put the activated version on the PATH for that session.

Reply:
Hi Carlton,

There is also the windows store version which may be better for beginners? I think it at least avoids the uncertainty of should it be ticked or not?

I'm not sure on `py` to pick a version. I'm a beginner and I've not really used `py`, so I don't know what I don't know. Also if we do go down the store root py launcher doesn't work. 

On a similar theme... Would there be value in a vscode / pycharm / other ide. I'm guessing most people will use something rather than 'just' writing it all in a standard text editor? 

Could point at something like this for example? 

https://code.visualstudio.com/docs/python/tutorial-django

Reply:
I think one option is fine. Let´s stick with the download for now. Drop add to path, and just show using the launcher. 

For reference: 

```
# Do this:
py -h

PS C:\Users\carlt> py --list
Installed Pythons found by C:\WINDOWS\py.exe Launcher for Windows
 -3.8-64 *
 -3.7-64
 -2.7-64

PS C:\Users\carlt> py -2
Python 2.7.17 (v2.7.17:c2f86d86e6, Oct 19 2019, 21:01:17) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>>

PS C:\Users\carlt> py -3.7
Python 3.7.5 (tags/v3.7.5:5c02a39a0b, Oct 15 2019, 00:11:34) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

i.e. pass it the number you want to use.

Reply:
Thanks. I'll make these amends. :-)